### ISSUE TITLE
[PHP] google.protobuf.Empty should be mapped to \Google\Protobuf\GPBEmpty

### ISSUE THREAD

**Issue by OmarElGabry** [grpc/grpc#21629]
If in the proto file we have a method like this:

```proto
rpc DeleteUser (DeleteUserRequest) returns (google.protobuf.Empty) {
        option (google.api.http) = {
            delete: "/users/{user_id=*}"
        };
    }
```

The (actual) generated php looks like this 
```php
public function DeleteUser(\Proto\User\DeleteUserRequest $argument,
      $metadata = [], $options = []) {
        return $this->_simpleRequest('/api.user.UserService/DeleteUser',
        $argument,
        ['\Google\Protobuf\Empty', 'decode'],
        $metadata, $options);
    }
```

The (expected) generated php looks like this 
```php
public function DeleteUser(\Proto\User\DeleteUserRequest $argument,
      $metadata = [], $options = []) {
        return $this->_simpleRequest('/api.user.UserService/DeleteUser',
        $argument,
        ['\Google\Protobuf\GPBEmpty', 'decode'],
        $metadata, $options);
    }
```

So, in summary, In case the class name is of type `\Google\Protobuf\Empty`, that should be mapped to [\Google\Protobuf\GPBEmpty](https://github.com/protocolbuffers/protobuf/blob/master/php/src/Google/Protobuf/GPBEmpty.php) instead. That's due to the fact that there is no "Empty" class in PHP (Empty is a reserved word in PHP), it is called "GPBEmpty". But that's how it is called in other languages like golang.

**Comment by stanley-cheung** [comment]
@OmarElGabry 
This issue seems to have been fixed in #13580. Can I ask whether you have compiled the latest `grpc_php_plugin` from `master`?

**Comment by OmarElGabry** [comment]
@stanley-cheung I am really sorry. I have been busy these days, and couldn't get back to you. Would you mind giving me some time, I will double-check and let you know. If you are 100% certain that this should have been solved by a previous fix, then feel free to close that issue. If not, I will check and let you know.
